servers/dsub/jobs/controllers/jobs_controller.py

```diff
@@ -257,13 +260,9 @@ def _provider_type():
     return current_app.config['PROVIDER_TYPE']
 
 
-def _query_result(job, project_id=None):
-    return QueryJobsResult(
-        id=job_ids.dsub_to_api(project_id, job['job-id'], job.get(
-            'task-id', '')),
-        name=job['job-name'],
-        status=job_statuses.dsub_to_api(job),
-        submission=job['create-time'],
-        start=job.get('start-time'),
-        end=job['end-time'],
-        labels=labels.dsub_to_api(job))
+def _query_jobs_response(jobs,
```

Comment:
opt_nit: This helper no longer has much value - I'd probably just inline it.

Reply:
Good idea, fixed.